Clarify when start tags can be marked self-closing in SVG/MathML (#44344)
mdn/content: files/en-us/glossary/void_element/index.md

Self-closing tags are required in void elements in {{Glossary("XML")}}, {{Glossary("XHTML")}}, and {{Glossary("SVG")}} (e.g., `<circle cx="50" cy="50" r="50" />`).

> In SVG and MathML, an element without any child nodes is allowed to be marked as self-closing by adding a trailing slash to its start tag. In such cases, if an element's start tag is marked as self-closing, the element must not have an end tag.

Reviewer: ```suggestion
In SVG and MathML, an element without any child nodes can be marked as self-closing by adding a trailing slash to its start tag. If an element's start tag is marked as self-closing, the element must not have an end tag.
```
